libs/design/src/molecules/list/list-subheader/list-subheader.component.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { DaffListSubheaderComponent } from './list-subheader.component';
+
+describe('DaffListSubheaderComponent', () => {
+  it('should create an instance', () => {
+    const component = new DaffListSubheaderComponent();
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Needs a test about adding css class to host.

Reply:
Resolved in 2ba76af